src/boot/efi/boot.c

```diff
@@ -2290,6 +2303,15 @@ static EFI_STATUS image_start(
         if (EFI_ERROR(err))
                 return log_error_status_stall(err, L"Error loading %s: %r", entry->loader, err);
 
+        if (entry->devicetree) {
+                err = devicetree_install(&dtstate, root_dir, entry->devicetree);
+                if (EFI_ERROR(err)) {
+                        Print(L"Error loading %s: %r\n", entry->devicetree, err);
```

Comment:
We have log_error_status_stall() for error logging now, see a couple lines above.